### PR TITLE
Stateless installation

### PR DESCRIPTION
**Pull request by tobiscr**
**Description**

Retrieve cluster state from Helm secrets instead using a ConfigMap which stores the cluster state.

**Related issue(s)**
https://github.com/kyma-project/kyma/issues/10297

### CODE REVIEW

**Comment by tobiscr** [comment]
Closing as too many conflicts will cause bigger changes

**Comment by tobiscr** [comment]
Corresponding PR on CLI side: https://github.com/kyma-project/cli/pull/814/files

**Review by strekm — APPROVED** [review]

File: function/go.mod [hunk 1]
```diff
@@ -10,6 +10,7 @@ require (
 	github.com/docker/docker v20.10.3+incompatible
 	github.com/docker/go-connections v0.4.0
 	github.com/docker/go-units v0.4.0 // indirect
+	github.com/docopt/docopt-go v0.0.0-20180111231733-ee0de3bc6815 // indirect
```
Comment (clebs) [inline function/go.mod:13]: How come we have this dependency added in an unrelated module?

Comment (tobiscr) [reply function/go.mod]: Happend when running `make build`

Comment (tobiscr) [reply function/go.mod]: Sorry, seems I accidentally accepted also the changes in the function mod... will revert them

Comment (tobiscr) [reply function/go.mod]: FIXED ... mod files of fct-module were reverted

**Pull request merged**